examples/all-clusters-app/all-clusters-common/all-clusters-app.zap

```diff
@@ -5605,16 +5605,16 @@
   "endpoints": [
     {
       "endpointTypeName": "Anonymous Endpoint Type",
-      "endpointTypeIndex": 0,
+      "endpointTypeIndex": 1,
       "profileId": "0x0103",
       "endpointId": 1,
       "networkId": 1
```

Comment:
Please change all `networkId` occurrences to `0` inside all `*.zap` files. This is likely the source of the `Cirque` failure. as I guess this is what prevent `Cirque` from running.

That's not your fault since `networkId: 1` is here since a long time, but generating `endpoint_id.h` instead of the old version from `AppBuilder` has exposed the issue.

I guess it brings us back to #1937 for the root issue.

I will approve the PR with this change ! :)

Reply:
Done !